### PR TITLE
Changes for the RHEL 8 distribution dependencies in the setup.md under Tutorials folder

### PR DESCRIPTION
**Pull request by phariyan**
There were several dependencies which couldn't be installed using 'yum install command' for the RHEL8 distribution while creating crio from source. The 'setup.md' file has been modified with a separate section added for the RHEL8 distribution which has the links for installing the dependencies using rpm(s).

### CODE REVIEW

**Comment by haircommander** [comment]
congrats on your first PR @phariyan :tada: A couple of requests:
You'll need to do `git fetch upstream` to make sure you're in sync with the master branch
Can you squash your commits with `git rebase -i upstream/master` to only have one commit? (you'll likely have to resolve merge conflicts also)
then you'll need to do git commit --amend -s to sign-off and stop the check from failing.

File: tutorials/setup.md [hunk 1]
```diff
@@ -39,6 +39,57 @@ yum install -y \
...
+yum install -y \
+  containers-common \
+  device-mapper-devel \
+  git \
+  glib2-devel \
+  glibc-devel \
+  glibc-static \
+  runc \
+```
+Use wget to download and run the following dependency:
```
Comment (haircommander) [inline tutorials/setup.md]: nit: s/dependency/dependencies

File: tutorials/setup.md [hunk 2]
```diff
@@ -39,6 +39,57 @@ yum install -y \
...
+RHEL 8 distributions:
+
+Make sure you are subscribed to the following repositories: \
+BaseOS/x86_64 \
+Appstream/x86_64 
+
+Follow the guide below to subscribe to the repositories if not already subscribed: \
+https://access.redhat.com/solutions/265523
+
+This required go version 12.1 or greater. Use the link below to install the appropriate go version for the system:\
```
Comment (umohnani8) [inline tutorials/setup.md]: nit: required to requires

File: tutorials/setup.md [hunk 3]
```diff
@@ -39,6 +39,57 @@ yum install -y \
...
+(Replace apt-get with yum for Linux based machines) \
```
Comment (umohnani8) [inline tutorials/setup.md]: Might want to rephrase this to say RHEL machines instead of Linux based, since that is a very broad term

File: tutorials/setup.md (hunk 2 above)
Comment (saschagrunert) [inline tutorials/setup.md]:
```suggestion
This required go version 1.12 or greater. Use the link below to install the appropriate go version for the system:\
```

**Comment by rhatdan** [comment]
@phariyan Please squash and sign your commits.

git rebase -i origin
git commit -a --amend -s
git push --force

**Comment by haircommander** [comment]
closing per https://github.com/cri-o/cri-o/pull/2483
